Cargo.toml

```diff
@@ -1,21 +1,34 @@
 [package]
 name = "sirius"
 version = "0.1.0"
+authors = [
+    "chaosma chao@snarkify.io",
+    "cyphersnake mikhail@snarkify.io",
+]
+repository = "https://github.com/snarkify/sirius"
+description = "A Plonkish folding framework for Incrementally Verifiable Computation (IVC)."
+keywords = [
+    "cryptography",
+    "ivc",
+    "nova",
```

Comment:
although we refer to Nova's implementation a lot, not sure if we should include nova here

Reply:
No, we should not.

Reply:
We should have the key word ZK and folding

Reply:
I just grabbed the tags from github. I guess I need to revisit both lists by doing a little research on which ones are the most popular.